client/src/components/StreamField/blocks/StreamBlock.js

```diff
@@ -69,145 +77,99 @@ class StreamBlockMenu extends BaseInsertionControl {
   constructor(placeholder, opts) {
     super(placeholder, opts);
     this.groupedChildBlockDefs = opts.groupedChildBlockDefs;
-    const animate = opts.animate;
 
     const dom = $(`
       <div>
-        <button data-streamblock-menu-open type="button" title="${h(
-          opts.strings.ADD,
-        )}"
-            class="c-sf-add-button c-sf-add-button--visible">
+        <button type="button" title="${comboBoxTriggerLabel}" class="c-sf-add-button c-sf-add-button--visible">
           <svg class="icon icon-plus" aria-hidden="true"><use href="#icon-plus"></use></svg>
         </button>
-        <div data-streamblock-menu-outer>
-          <div data-streamblock-menu-inner class="c-sf-add-panel"></div>
-        </div>
       </div>
     `);
     $(placeholder).replaceWith(dom);
     this.element = dom.get(0);
-
-    this.addButton = dom.find('[data-streamblock-menu-open]');
-    this.addButton.click(() => {
-      this.toggle();
-    });
-
-    this.outerContainer = dom.find('[data-streamblock-menu-outer]');
-    this.innerContainer = dom.find('[data-streamblock-menu-inner]');
-    this.hasRenderedMenu = false;
-    this.isOpen = false;
+    this.addButton = dom.find('button');
+    this.combobox = document.createElement('div');
     this.canAddBlock = true;
     this.disabledBlockTypes = new Set();
```

Comment:
I’ve decided to retain this tracking of disabled block types as-is, even though the UI currently does nothing with i. This will be used again once we display the min/max block count (right now our current designs only show max).